meta/runtime.yml

```diff
@@ -2,301 +2,301 @@ plugin_routing:
   modules:
     azure_rm_aks_facts:
       deprecation:
-        removal_date: TBD
+        removal_version: 3.0.0
```

Comment:
I think I would change all removal versions to 2.0.0. Since there's no plans for even a 2.0.0, 3.0.0 would be "never". 2.0.0 can be released eventually (maybe for Ansible 5.0.0?) and then the deprecated stuff is gone.